phpBB/phpbb/files/filespec_storage.php

```diff
@@ -288,6 +288,10 @@ public function remove($storage)
 		{
 			$storage->delete($this->destination_file);
 		}
+		else
+		{
+			@unlink($this->filename);
```

Comment:
@Nicofuma Check this, i am not sure about it, becuase this wasn't on the original filespec.php

Reply:
when will we hit this case?
I don't think it is harmeful

Reply:
Ok, i will remove it, it makes the tests fail